base_layer/core/src/transactions/transaction_components/encrypted_data.rs

```diff
@@ -159,8 +158,6 @@ pub enum PaymentId {
     /// No payment ID.
     #[default]
     Empty,
-    /// A u64 number.
-    U64(u64),
```

Comment:
We must still be able to decrypt a `PaymentId::U64` from the genesis block - this is used in pre-mine spending